whats_fresh/whats_fresh_api/templates/vendor.html

```diff
@@ -112,7 +137,101 @@ <h3 class="title">Products</h3>
 </div>
 {% endblock content %}
 {% block footer %}
+    <script src="https://code.jquery.com/jquery-1.10.2.min.js"></script>
```

Comment:
We're already calling jquery 2.1.1. Try removing this and seeing if the map and related functionality still works. It probably won't, but it'd be easier than trying to use both versions cleanly.